src/tests/ftest/cart/no_pmix_group_test.c

```diff
@@ -686,7 +686,7 @@ int main(int argc, char **argv)
 	}
 
 	for (i = 0; i < 10; i++) {
-		rc = asprintf(&uris[i], "ofi+tcp;ofi_rxm://127.0.0.1:%d",
+		rc = asprintf(&uris[i], "ofi+tcp://127.0.0.1:%d",
```

Comment:
i dont think it matters here as these are just bogus uris to test group operations later on. group addition requires a rank and uri, but we dont check it for any validity i dont think.
otherwise we should modify this test and read out provider used